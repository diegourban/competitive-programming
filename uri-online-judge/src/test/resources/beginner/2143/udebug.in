5
17
32
9
21
3
3
6
10
35
5
51
52
34
65
60
0