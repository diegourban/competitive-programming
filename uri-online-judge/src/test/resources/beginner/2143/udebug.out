33
62
17
41
5
10
18
69
101
102
66
129
118
